### DISCUSSION TITLE
Website Refresh Notifications

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11069] [General - Components / Directives / etc] [answered]
I was wondering if anyone else was experiencing this. But it seems like every single day i go to the Quasar website, it prompts me with the a red 'Refresh Notification' at the top of the webpage. It shows almost everyday. Could someone explain what triggers this to appear at such a high frequency. If I'm being blunt about it, it's quite annoying. I used to hit 'Refresh' all the time but then hit 'Dismiss' and neither seem to mitigate the popup.

**Comment by MilosPaunovic** [comment, marked as answer]
What triggers it is a new release of Quasar. There were a few ones in the last couple of days including some bugfixes.

https://github.com/quasarframework/quasar/releases

**Reply by JokerMartini** [reply]
I just noticed when im on the website and click 'Refresh' i can then navigate to another page on the same site and it will notify me again to refresh. It seems like it has a unique Refresh per page, rather than a single refresh across the website. I should have explained that better.

**Comment by rstoenescu** [comment]
We're sorry for working so hard on Quasar and releasing bugfixes and features almost daily :) :P

**Reply by JokerMartini** [reply]
haha i just wanted to make sure i wasn't experiencing something that was a bug itself. :) im all for the bug fixes. Thanks for all the hard work.

**Reply by JokerMartini** [reply]
I just noticed when im on the website and click 'Refresh' i can then navigate to another page on the same site and it will notify me again to refresh. It seems like it has a unique Refresh per page, rather than a single refresh across the website. I should have explained that better.
